src/components/Cards/MountainInfoCard.tsx

```diff
@@ -62,25 +45,53 @@ const MountainInfoCard = () => {
         >
             <Typography style={{ fontSize: 32, fontWeight: 500 }}>Mountain</Typography>
             <Box pt={2} flexGrow={1} display={"flex"} flexDirection={"column"}>
-                <Box display={"flex"} justifyContent={"space-around"}>
-                    <Box display={"flex"} flexDirection={"column"} alignItems={"center"}>
-                        <Typography style={{ fontSize: 24, fontWeight: 500 }}>
-                            Trails Open
-                        </Typography>
-                        <TrailInfo totalOpenTrails={totalOpenTrails} totalTrails={totalTrails} />
-                    </Box>
-                    <Box display={"flex"} flexDirection={"column"} alignItems={"center"}>
-                        <Typography style={{ fontSize: 24, fontWeight: 500 }}>
-                            Lifts Open
-                        </Typography>
-                        <LiftsInfo totalOpenLifts={totalOpenLifts} totalLifts={totalLifts} />
-                    </Box>
+                {snowfallData && snowReport && mountainAreas ? (
```

Comment:
We should put this kind of check in each graph so that some could load independently from the others.